examples/exotica_examples/resources/configs/example_eff_axis_alignment.xml

```diff
@@ -38,10 +38,9 @@
       <Task Task="Position" Rho="1"/>
       <Task Task="Direction" Rho="1"/>
     </Cost>
-
-    <StartState>0 0 0 0 0 0 0</StartState>
-    <NominalState>0 0 0 0 0 0 0</NominalState>
```

Comment:
Sorry I should have been more clear: 

1. You can remove StartState if it is all zeros. That will be the default.
2. If you remove NominalState, using null space will be _turned off_. Setting NominalState to zero will activate null space for solvers that support it (e.g., pseudo-inverse IK). As such removing L43 changed the functionality of this demo [not sure whether that was intended].

Reply:
I don't think it makes much difference having `<NominalState>` there. The goal of the example script, I think, should to be to highlight how `EffAxisAlignment` can be used and how the task map specifically works. I feel a general explanation of `<NominalState>` should exist elsewhere in the documentation.